pathspider/plugins/ecnspider3.py

```diff
@@ -155,6 +169,47 @@ def create_observer(self):
             traceback.print_exc()
             sys.exit(-1)
 
+    def combine_flows(self, flow):
+        dip = flow['dip']
+        if dip in self.comparetab:
+            other_flow = self.comparetab.pop(dip)
```

Comment:
Should we care about two connections to the same host happening (i.e. two sets of connections making two with ECN and two without ECN in this case), maybe with different hostnames being used or some other application layer change? Perhaps not relevant for ECN testing, but other plugins might need to match differently?